### PR TITLE
CC senior delegates for competition confirmation emails.

### PR DESCRIPTION
**Pull request by jfly**
Pretty straightforward. We should have been doing this from the start.

### CODE REVIEW

**Comment by jfly** [comment]
To avoid shocking delegates with this change, we're going to wait for a Board member to tell us to merge this up before merging this up.

File: WcaOnRails/app/mailers/competitions_mailer.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ def notify_board_of_confirmed_competition(confirmer, competition)
     @confirmer = confirmer
     mail(
       to: "board@worldcubeassociation.org",
-      cc: competition.delegates.pluck(:email),
+      cc: competition.delegates.pluck(:email) + competition.delegates.map { |d| d.senior_delegate&.email }.compact.flatten(1),
```
Comment (larspetrus) [inline WcaOnRails/app/mailers/competitions_mailer.rb]: 1. `flatten' looks weird. Can `senior_delegate&.email` really be an array?

2. But several delegates can probably have the same senior delegate. So `.compact.uniq` is maybe the right way?

Comment (jfly) [inline WcaOnRails/app/mailers/competitions_mailer.rb]: Whoops, you're right, that flatten is pointless! I'll change it to a `compact.uniq` =)

Comment (jfly) [inline WcaOnRails/app/mailers/competitions_mailer.rb]: Ok, updated with a test!

**Comment by larspetrus** [comment]
LGTM!

**Comment by jfly** [comment]
The Board is in favor of this, and @pedrosino has already notified the delegates, so I'm merging this up!

**Pull request merged**